src/common.css

```diff
@@ -0,0 +1,413 @@
+    font-weight: 700;
+}
+
+.chat-active p:nth-child(1){
+    font-weight: 700;
+}
+
+.chat-text p:nth-child(2){
+    color: var(--secondary-gray);
+    margin-right: 40px;
+}
+
+.dot{
+    width: 5px;
+    height: 5px;
+    border-radius: 100px;
+}
+
+.dot-chat-notification{
+    background-color: var(--primary-blue);
+}
+
+.dot-course-notification{
+    background-color: var(--primary-red);
+}
+
+.dot-settings{
+    background-color: var(--secondary-gray);
+}
+
+.small{
+    /* --size: 30px; */
+}
+
+/* INPUT FIELDS */
+.input-field{
+    display: flex;
+    align-items: center;
+    gap: 20px;
+    padding: 0 20px;
+    border: 1px solid var(--tertiary-gray);
+    border-radius: 10px;
+    justify-content: space-between;
+}
+
+.input-field input[type="text"]{
+    border: none;
+    line-height: 3;
+    font-size: 15px;
+}
+
+
+.input-field input:focus{
+    outline: none;
+}
+
+.input-field input:focus{
+
+}
+
+.anonymous{
+    display: flex;
+    align-items: center;
+    gap: 10px;
+}
+
+/* INPUT CONTAINER */
+.input-header{
+    text-align: left;
+    color: var(--primary-blue);
+}
+
+.input-wrapper{
+    box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.1);
+    padding: 40px 30px;
+    border-radius: 10px;
+    display: grid;
+    row-gap: 40px;
+    width: min-content;
+    margin: 0 auto;
+}
+
+.input-container{
+    display: grid;
+    text-align: left;
+}
+
+.input-container h3{
+    color: var(--primary-blue);
+    margin-bottom: 5px;
+}
+
+.input-container input{
+    border: 1px solid var(--tertiary-gray);
+    border-radius: 10px;
+    padding: 10px;
+    outline: none;
+    font-size: 14px;
+}
+
+.input-btn{
+    justify-content: space-between;
+    display: flex;
+    align-items: center;
+}
+
+
+/* CONTENT CONTAINER */
+.content-container{
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    height: 100vh;
+    text-align: center;
+    row-gap: 50px;
+}
+
+/* DROP DOWN */
+.dropdown{
+    background-color: #fff;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: var(--box-shadow);
+}
+
+.dropdown .top-bar{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 5px;
+}
+
+.dropdown ul{
+    color: var(--primary-gray);
+}
+
+.dropdown li:hover{
+    color: var(--primary-blue);
+    font-weight: 700;
+    cursor: pointer;
+}
+
+/* SETTINGS ICON */
+
+.settings-icon{
+    display: flex;
+    gap: 3px;
+}
+
+/* POST FLAG */
+.post-flags{
+    display: flex;
+    row-gap: 10px;
+    flex-direction: column;
+    align-items: flex-end;
+}
+
+.post-flag{
+    border-radius: 10px 0 0 10px;
+    background-color: #fff;
+    box-shadow: var(--box-shadow);
+    width: fit-content;
+    transition: var(--transition);
+    cursor: pointer;
+}
+
+.post-flag-small:hover{
+    padding: 20px;
+    color: var(--primary-blue);
+}
+
+.post-flag-large{
+    padding: 20px;
+}
+
+.post-flag-small{
+    color: var(--secondary-blue);
+    padding: 10px;
+}
+
```

Comment:
After some time of giving feedback to this file, it became chaotic.
And the reason for that, is because you have almost all your CSS in a single file.
That is neither maintainable nor readable. It would be essential to split it up to their components.
You have created CSS files for your components and they are empty. Thus, you need to do some cleanup.